File: DownloadFiles/TGNet/Chapter06/VB/Exercise6-5/TestHarness/TestHarness/Form1.vb
Public Class Form1

  Private Sub btnEmptyString_Click _
  (ByVal sender As System.Object, ByVal e As System.EventArgs) _
  Handles btnEmptyString.Click

    If Validations.Strings.TestForEmptyStrings(TextBox1.Text) = True Then
      MessageBox.Show("it was empty")
    Else
      MessageBox.Show("it was not empty")
    End If

  End Sub
End Class

File: DownloadFiles/TGNet/Chapter06/C#/Exercise6-5/TestHarness/TestHarness/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace TestHarness
{
  public partial class Form1 : Form
  {
    public Form1()
    {
      InitializeComponent();
    }

    private void btnEmptyString_Click(object sender, System.EventArgs e)
    {
      if (Validations.Strings.TestForEmptyStrings(textBox1.Text) == true)
      {
        MessageBox.Show("it was empty");
      }
      else
      {
        MessageBox.Show("it was not empty");
      }
    }
  }
}